### DISCUSSION TITLE
BNote 4.0.3 beta ready for testing

### DISCUSSION THREAD

**Discussion by mattimaier** [mattimaier/bnote#555] [Announcements]
Here are two versions with updated libaries for BNote 4.0.2 and a full one for installation from scratch:
* [4.0.3 Patch Beta](https://github.com/mattimaier/bnote/raw/refs/heads/master/BNote-Releases/BNote/Patches/bnote_patch403beta.zip)
* [4.0.3 Full](https://github.com/mattimaier/bnote/raw/refs/heads/master/BNote-Releases/BNote/Patches/bnote_release-403beta_full.zip)

**Comment by hermannluehrs** [comment]
Hab auf strato/mariaDB10 neu sukzessive 4.0/4.01/4.02/4.03 installiert und bisher testdatensätze (kontakte, proben, auftritte, termine) angelegt: ohne fehler. einzige mankos: 
- modul share: Pfadanzeige fehlt bei geschachtelten Ordnern zB im Bereich Mitglieder 
- Zurück-Button ist sehr klein zum Klicken
- modul startseite: Detailansicht von Terminen liefert nur die Informationen, die auch in der Eingangspalte stehen. Es fehlen die Informationen (zB Notizen), die in der Detailsicht im Kalender zu sehen sind.

**Comment by hermannluehrs** [comment]
Die Filterfunktion auf der Startseite für Proben etc ist supi! Wäre noch feiner, wenn man in der Konfiguration die max. Anzahl der Anzeigen jeweils gestlegen kann. Das ist ja angekündigt (vieleicht auf der Startseite noch mit button 'zeige alles')

**Comment by hermannluehrs** [comment]
Installation von 4.03 (beta_full) auf strato/mysql8 läuft. einige testdatensätze zeigen fehler, die auf mysql 5.7 oder auch mariaDB10 nicht vorkommen:
- Titelliste für Programm wird nicht gespeichert
- Modul Mitglieder zeigt Kacheln nicht an, wenn der Kontakt ncht Administrator ist
- Modul Konfiguration: Obwohl Benachrichtigungen deaktiviert, werden Mails bei Kommentaren versendet.
